test/extended/networking/endpoint_admission.go

```diff
@@ -29,10 +30,9 @@ var _ = g.Describe("[sig-network][endpoints] admission", func() {
 	})
 })
 
-var exampleAddresses = map[string]string{
-	"cluster":  "10.128.0.2",
-	"service":  "172.30.0.2",
-	"external": "1.2.3.4",
+var exampleServiceAddresses = map[string]string{
```

Comment:
@ardaguclu I would move this into the TestEndpointAdmission() function since it's only used there and it's confusing to have it as a global when testOne() also creates a map of the same name, but in local scope.

Reply:
In fact, why even bother with the map? Why not just assign the addresses directly since exampleServiceAddresses is only used in one place?

Reply:
Thanks for review @dcbw, I changed according to your suggestions.